src/Entities/Player.cpp

```diff
@@ -1702,6 +1702,23 @@ void cPlayer::SendRotation(double a_YawDegrees, double a_PitchDegrees)
 
 
 
+void cPlayer::SpectateEntity(cEntity * a_Target)
```

Comment:
This should either accept `nullptr` or `a_Target` should be a reference. As written, a plugin could crash the server by doing `Player:SpectateEntity(nil)`.